sockeye/evaluate.py

```diff
@@ -90,7 +93,7 @@ def main():
             scores = []
             for metric in args.metrics:
                 if metric == C.BLEU:
-                    bleu = raw_corpus_bleu(h, r, args.offset)
+                    bleu = raw_corpus_bleu([h], [r], args.offset)
                     scores.append("%.6f" % bleu)
                 elif metric == C.CHRF:
                     chrf_score = raw_corpus_chrf(h, r)
```

Comment:
I assume this is the bug? This is really a problem with our interface because this mistake is too easy to make. What do you think about adding some stricter type checking? Maybe the problem is unavoidable because I'm not sure that we want to disallow passing in strings, but that might be the better option.

Reply:
We could do some stricter type checking but ideally we shouldn't call the function `corpus_bleu` anyway if we want to compute sentence BLEU. My suggestion would be to add a `sentence_bleu` function in sockeye/evaluate.py that calls sacrebleu BLEU code to compute (properly smoothed) sentence-level BLEU scores.